lib/msal-browser/src/interaction_client/RedirectClient.ts

```diff
@@ -114,6 +115,16 @@ export class RedirectClient extends StandardInteractionClient {
             validRequest.loginHint || "",
             validRequest.account || null
         );
+
+        // Cache thumbprints for telemetry linking if PerformanceClient is not 3P
```

Comment:
`Read thumbprints from temporary cache if PerformanceClient is not 3P` -> `Read thumbprints from temporary cache if PerformanceClient is instance of BrowserPerformanceClient`. Please remove this comment as code below is pretty clear